netbout-client/src/test/java/com/netbout/client/RtFriendsITCase.java

```diff
@@ -109,4 +109,19 @@ public void throwsExceptionIfInvalidToken() throws Exception {
         final String friend = "bob";
         bout.friends().invite(friend);
     }
+
+    /**
+     * RtFriends can throw an exception if unknown alias kicked.
+     * @throws Exception If there is some problem inside
+     */
+    @Test (expected = Friends.UnknownAliasException.class)
+    public void throwsExceptionIfUnknownAliasKicked() throws Exception {
+        final User user = this.rule.get();
```

Comment:
@dmzaytsev We should be able to inline `user`, `alias`, and `inbox`, since they are just one-use variables.